javaparser-core-generators/src/main/java/com/github/javaparser/generator/core/CoreGenerator.java

```diff
@@ -11,28 +12,37 @@
 
 /**
  * Generates all generated visitors in the javaparser-core module.
+ * Suggested usage is by running the run_core_generators.sh script.
+ * You may want to run_metamodel_generator.sh before that.
  */
 public class CoreGenerator {
+    private static final ParserConfiguration parserConfiguration = new ParserConfiguration()
+//                                .setStoreTokens(false)
```

Comment:
Why do we have stuff commented out in the file?

Reply:
See #1301